src/containers/DeveloperPortal/containers/SdkAndLibraries/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -0,0 +1,25 @@
+@import 'src/styles/mixins';
+@import 'styles/font-mixins';
+@import '../../shared';
+
+.Breadcrumb {
```

Comment:
will naming it as `Breadcrumb` be conflicting to any other existing `Breadcrumb` classnames?

Reply:
You're right! But right now, these three components have the exact same styling. So, overwriting won't break anything.

I think this will be solved automatically once we move out the dev portal or migrate to styled components.

If you've noticed any major styling issue I can rename these, otherwise we can roll with it for now.

Reply:
hmm I see, if the 3 components have the exact same styling, maybe we have a common shared scss file. But yeah, maybe can be done when we migrate to styled components